tools/hlstool/hlstool.cpp

```diff
@@ -349,6 +360,61 @@ static LogicalResult doHLSFlowDynamic(
   return success();
 }
 
+static LogicalResult doHLSFlowCalyx(
+    PassManager &pm, ModuleOp module,
+    std::optional<std::unique_ptr<llvm::ToolOutputFile>> &outputFile) {
+
+  if (irInputLevel < 0) {
+    irInputLevel = HLSFlowDynamicIRLevel::High; // Default to highest level
+  }
+  if (irOutputLevel < 0) {
+    irOutputLevel = HLSFlowDynamicIRLevel::Sv; // Default to lowest level
+  }
+
+  if (irInputLevel != HLSFlowDynamicIRLevel::High) {
+    llvm::errs() << "Error: Calyx flow only supports 'high' IR input\n";
```

Comment:
Nit: (here and above) this "Error:" in the beginning of the string seems redundant.

Reply:
Thanks, fixed!